src/js/utils/language.js

```diff
@@ -0,0 +1,51 @@
+define([
+], function () {
+
+    /**
+     * A map of 2-letter language codes (ISO 639-1) to language name in English
+     */
+    var twoCharMap = {
+        'zh': 'Chinese',
+        'nl': 'Dutch',
+        'en': 'English',
+        'fr': 'French',
+        'de': 'German',
+        'it': 'Italian',
+        'ja': 'Japanese',
+        'pt': 'Portuguese',
+        'ru': 'Russian',
+        'es': 'Spanish'
+    };
+
+    /**
+     * A map of 3-letter language codes (ISO 639-2) to 2-letter language codes (ISO 639-1)
+     */
+    var threeCharMap = {
```

Comment:
I'd like to hold back on this map until we find the need to support ISO 639-2 codes.